doc/rbd/libvirt.rst

```diff
@@ -247,7 +247,7 @@ commands, refer to `Virsh Command Reference`_.
 
 #. Get the ``client.libvirt`` key and save the key string to a file. ::
 
-	ceph auth get-key client.libvirt | sudo tee client.libvirt.key
+	ceph auth get-key client.libvirt | sudo tee -i client.libvirt.key
```

Comment:
@dennisTGC I am not sure I understand.

As I see `-i` (`--ignore-interrupts`) option just makes `tee` to ignore `Ctr+C`, which a user might want to send to interrupt the process that provides input (`ceph auth get-key` in this case) and wants `tee` to exit gracefully. I don't see how it may be related to "escape characterts" you are talking about in the commit description.